package.json

```diff
@@ -56,6 +56,7 @@
     "acorn": "^8.8.2",
     "acorn-import-attributes": "^1.9.5",
     "cjs-module-lexer": "^1.2.2",
-    "module-details-from-path": "^1.0.3"
+    "module-details-from-path": "^1.0.3",
+    "@babel/helper-validator-identifier": "^7.24.7"
```

Comment:
I'd like to hear from @Qard or @bengl what they think about adding in this dependency.

Reply:
Bump

Reply:
We could vendor this code and it's around 13kB.

This package has 50m weekly downloads so there's a high chance it will already be in many users dependencies.

Reply:
> This package has 50m weekly downloads so there's a high chance it will already be in many users dependencies.

Maybe. But I've never seen the package before this PR. I looked through it back when this PR was opened, and I remember getting the impression that it could be overkill. So I wanted an opinion from the folks much more in tune with this code than I am.

Reply:
I'm fine with it, though I wonder if we really need _all_ that and not just a simplified subset of it.

Reply:
We need all of `identifier.js` (12.2kB) which is most of the package.

<img width="787" alt="image" src="https://github.com/user-attachments/assets/ce5fa55b-b346-43f0-91b1-587f0bbfbaa1">